fgpyo/sam/__init__.py

```diff
@@ -195,6 +197,12 @@
 NO_REF_POS: int = -1
 """The reference position to use to indicate no position in SAM/BAM."""
 
+STRING_PLACEHOLDER: str = "*"
+"""The value to use when a string field's information is unavailable."""
```

Comment:
If you place this above line 194, you can use it on the right-hand-side of `NO_REF_NAME`. I don't think we need this variable, but I also don't mind having it.